deepspeed/ops/transformer/transformer.py

```diff
@@ -405,6 +408,25 @@ def backward(ctx, grad_output):
              norm_w,
              norm_b)
 
+        # This appears to be an effective way to release context memory
```

Comment:
Did this have any performance implication at all!?

Reply:
No, I did not observe any perf impact except for the memory savings.

Reply:
this is great, thanks Tunji for investigating this :)